content/_changelogs/10.11.0.md

```diff
@@ -0,0 +1,32 @@
+## 10.11.0
+
+_Released 10/25/2022_
+
+**Features:**
+
+- Adds support for [`cy.session()`](/api/commands/session) to experimental WebKit.
+Ensures that with `experimentalSessionAndOrigin` enabled, tests are still able to run against webkit. Addresses [#24116](https://github.com/cypress-io/cypress/issues/24116), [#23832](https://github.com/cypress-io/cypress/issues/23832).
+- Improves user experience when connecting a project to the Dashboard. Addresses [#23379](https://github.com/cypress-io/cypress/issues/23379), [#23766](https://github.com/cypress-io/cypress/issues/23766), [#23767](https://github.com/cypress-io/cypress/issues/23767).
+  - Users can now connect a project to the Dashboard when logging in from the Cypress Launchpad after a testing type has been selected.
```

Comment:
This isn't a bug fix?

Reply:
The `Users can now connect a project to the Dashboard when logging in from the Cypress Launchpad after a testing type has been selected.` is a new feature.  I will rephrase to make that more clear.

Reply:
Updated